src/localization/en.js

```diff
@@ -222,6 +222,7 @@ export default {
           childSupport:     ` child support`,
         },
```

Comment:
Can you get rid of this snippet please, now that it's replaced with `childSupportExpense_v1`?